smarc_bringup/launch/auv_scenarios.launch

```diff
@@ -1,7 +1,8 @@
 <launch>
 
 	<!-- Gazebo environment -->
-	<arg name="gazebo_env" default="random_rocks"/>
+	<arg name="gazebo_env" default="pipe_following"/>
```

Comment:
This should be part of a separate launch file that calls this. Like `pipe_following_demo.launch`.